tests/rptest/tests/restart_services_test.py

```diff
@@ -49,21 +49,20 @@ def test_restart_services(self):
         # Failure checks
         self.logger.debug("Check restart with no service name")
         try:
-            admin.redpanda_services_restart()
+            admin.restart_service()
```

Comment:
I feel like this check, and the ones below, should have an `assert false` following the call as we expect these calls to throw exceptions.

Reply:
I don't think this is a blocker, could be one of those test improvement things we should get back to

Reply:
Good idea.

This PR is for test improvements so I'll do it now.